StreetHawk/Classes/Location/Internal/SHLocationManager.m

```diff
@@ -756,6 +762,8 @@ - (void)locationManager:(CLLocationManager *)manager didFailWithError:(NSError *
         if (StreetHawk.currentInstall != nil && (sentFlag == nil || sentFlag.length == 0))
         {
             [StreetHawk sendLogForCode:LOG_CODE_LOCATION_DENIED withComment:@"Location service denied by user."];
+            [StreetHawk tagString:@"true" forKey:@"sh_location_denied"];
+            SHLog(@"sh_location_denied set as true due to Location service denied by user");
```

Comment:
You tag does same thing as LOCATION_DENIED_SENT logline. Consider removing the logline?